matchbox/fixture_test.go

```diff
@@ -0,0 +1,245 @@
+package matchbox
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"net"
+
+	"google.golang.org/grpc"
+
+	"github.com/coreos/matchbox/matchbox/rpc"
+	"github.com/coreos/matchbox/matchbox/server"
+	"github.com/coreos/matchbox/matchbox/storage"
+)
+
+type FixtureServer struct {
+	Store    storage.Store
+	Server   *grpc.Server
+	Listener net.Listener
+}
+
+func NewFixtureServer(s storage.Store) *FixtureServer {
+	return &FixtureServer{
+		Store: s,
+	}
+}
+
+func (s *FixtureServer) Start() {
+	var err error
+	s.Listener, err = net.Listen("tcp", "127.0.0.1:0")
```

Comment:
See the other note, just use a string constant 127.0.0.1:8081 to match the default docs. Its fine. Then you don't need to hold the listener in a struct just to call s.Listener.Addr().String() later.